DESCRIPTION

```diff
@@ -39,6 +38,8 @@ Imports:
     base64enc,
     shiny,
     rstudioapi
+Remotes:
+    dplyr (>= 0.5.0.9000)
```

Comment:
Thanks, we get now a `Namespace dependency not required` in `R CMD check`, but If I remove the `@import dplyr` refs we get an error referencing `collect` from `dplyr` 😕